WISE-Community/WISE-Client
# feat(Dialog Guidance): Implement computer avatars

## Changes
Add computer avatar to Dialog Guidance authoring
Add computer avatar selection to Dialog Guidance in student VLE

## Test
Authoring
- Enabling computer avatar in Dialog Guidance
- Authoring computer avatar fields (label, prompt, initial response) in Dialog Guidance
- Selecting which computer avatars to make available to students in Dialog Guidance

Student
- Make sure the student can select a computer avatar
- Make sure the computer avatar fields like label, prompt, and initial response show up properly
- Make sure choosing a computer avatar, leaving the step, and coming back, will re-populate the chosen computer avatar
- Make sure Dialog Guidance still works when computer avatar is not enabled

Grading
- Make sure computer avatars show up in Dialog Guidance grading view
